flatsurf/geometry/translation_surface.py

```diff
@@ -638,6 +694,8 @@ def left(self, label):
     def _repr_(self):
         return "Origami defined by r=%s and u=%s"%(self._r,self._u)
 
+
+
```

Comment:
```suggestion
```

Reply:
I thought that Python recommands 2 linebreaks between class definitions. (at least this is what pyflakes suggests by default)